{"uuid":"d0dac1d9-cc80-45a7-8c4a-0628f31de352","historyId":"83f2e4e7578f72de03a371522fcf04c4","fullName":"cucumber.api.testng.AbstractTestNGCucumberTests.runScenario","labels":[{"name":"package","value":"api.runner.CucumberRunner"},{"name":"testClass","value":"api.runner.CucumberRunner"},{"name":"testMethod","value":"runScenario"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Vendor"},{"name":"subSuite","value":"api.runner.CucumberRunner"},{"name":"host","value":"FNAVDev119"},{"name":"thread","value":"10116@FNAVDev119.main(1)"}],"links":[],"name":"Runs Cucumber Scenarios","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"{\n    \"result\": false,\n    \"statusCode\": 0,\n    \"errorMessage\": \"The remote server returned an error: (400) Bad Request.\",\n    \"dateCompleted\": \"01/02/2019\"\n} expected [true] but found [false]","trace":"java.lang.AssertionError: {\n    \"result\": false,\n    \"statusCode\": 0,\n    \"errorMessage\": \"The remote server returned an error: (400) Bad Request.\",\n    \"dateCompleted\": \"01/02/2019\"\n} expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:96)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\r\n\tat org.testng.Assert.assertTrue(Assert.java:44)\r\n\tat utils.Helper.validateResponseBody(Helper.java:71)\r\n\tat api.steps.AssertionSteps.response_contains_in_any_order(AssertionSteps.java:31)\r\n\tat âœ½.response includes the following in any order(src/test/resources/api/features/CompleteTireEvent.feature:24)\r\n"},"stage":"finished","start":1546436769549,"stop":1546436773705,"parameters":[{"name":"arg0","value":"\"Complete Tire Event - /api/vmEvent/completeTireEvent\""},{"name":"arg1","value":"\"Complete Tire Event\""}]}